<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook cb8">4.68 <span class="authors"></span></div>
</div></header>





## Given   
A spring-loaded piston-cylinder device contains 1 kg of carbon dioxide. This system is heated from 100 kPa and 25°C to 1000 kPa and 300°C. 

![figure, cb8, 4-69](../figures/ch04_cen98179_p04068_lg.jpg)

## Find 
1. $Q_{in}$   
1. $W_{out}$

## Get Organized



### These things you should notice right away:

1. Closed system ⋮ constant mass  
1. Ideal Gas
1. No kinetic energy considerations
1. The only possible potential energy cosideration would be associatedd with the weight of the piston.
1. No information about piston mass, piston area, or spring constant.
1. All work is boundary work.



We seek $Q_{in}$ and $W_{out}$, which appear in our energy equation as follows:

\begin{gather}
  E_\ce{in} &-& E_\ce{out} &=& \Delta E_\ce{sys} \\
  Q_\ce{in} &-& W_\ce{out} &=& U_2 - U_1 \\
\end{gather}

What do we know?  

1. The interal energy of an ideal gas is a function of temperature alone, so we can get $U_1$ and $U_2$.
2. All our work is boundary work, so we can calculate the work without using the energy equation.
3. After we calculate work, we can use the energy equation to calculate heat transfer.

# Attack

1. Get $ U_1, U_2$  

  We can get thermodynamic properties of $\ce{CO2}$ from the ideal gas tables in the textbook. You should notice that the values are given on a molar basis. No problem:

  \begin{equation}
   u = \frac{\bar u}{\textbf{MW}}
  \end{equation}

In [1]:
using Unitful
MW_CO₂ = 44.01u"kg/kmol"  # table

ū1 = 6885u"kJ/kmol"       # table
ū2 = 16251.96u"kJ/kmol"   # table

u1 = ū1 / MW_CO₂
u2 = ū2 / MW_CO₂

@show u1 u2;

u1 = 156.44171779141104 kJ kg⁻¹
u2 = 369.2788002726653 kJ kg⁻¹


----

2. Calculate $W_b$

  We have to resolve a spring force. Let's start with a free-body diagram of our piston,

  ![figure, cb8, 4-69, free body diagram](../figures/cb8,4-068fbd.png)
  
   and the general force balance that mathematically describes it.

  \begin{gather}
  \begin{gathered}
    F_\ce{sys} &-& F_\ce{piston} &-& F_\text{spring} &-& F_\ce{atm} &=& 0 \\ 
    P_\ce{sys}A_\ce{piston}
    &-& m_\ce{piston}g
    &-& F_\text{spring}
    &-& P_\ce{atm}A_\ce{piston} &=& 0 
  \end{gathered} \label{eq:forcebalance} \\
    P_\ce{sys}\underline{A_\ce{piston}} 
    = \underline{m_\ce{piston}g} 
    + F_\text{spring}
    + \underline{P_\ce{atm}A_\ce{piston}}  
  \end{gather}

  All of the underlined terms are constants. With a quick differentiation, 

  \begin{gather}
    {\rm d}P_\ce{sys}A_\ce{piston} = 
    {\rm d}F_\text{spring} \\
  \end{gather}

   we conclude that our system pressure varies linearly with spring force. By Hooke's Law, we know spring force varies linearly with displacement. The constant-area piston means displacecment varies linearly with volume. Mathematically, 
   
  \begin{gather}
    \Delta P \propto \Delta F_\text{spring} \propto \Delta z \propto \Delta V
  \end{gather}

  Therefore, our system pressure varies linearly with volume. 
  
  <img alt="pv diagram" src="../figures/cb8,4-068pvdiagram_filled.png" width=50%>
  
  Boundary work $ W_{b,out} = \int P\,{\rm d}V$, the area under the curve of our process on a P-V diagram. Integrate it, either algebraically or graphically. It's a trapezoid; I like that route:
  
  $$ W_{b,out} = \int^{V_2}_{V_1} P\,{\rm d}V 
         =\textstyle \frac{1}{2}\big(P_1 + P_2 \big)\big( V_2 - V_1 \big) $$
         
  Remember, it's an ideal gas:
         
  $$ PV = mRT \qquad 
   \ce{R_\ce{CO2}} = \frac{\overbar{\rm R}}{\textbf{MW}_{\ce{CO2}}} $$

  \begin{gather}
    W_{b,out} = \displaystyle \frac{1}{2}\bigg(P_1 + P_2 \bigg)m\ce{R_\ce{CO2}}\bigg( \frac{T_2}{P_2} - \frac{T_1}{P_1} \bigg)
  \end{gather}







In [2]:
m = 1u"kg"          # given
R̄ = u"R"            # Unitful built-in constant
R_CO₂ = R̄/MW_CO₂    # engineering gas conostant

P1 = 100u"kPa"         # given
P2 = 1u"MPa"           # given
T1 = 25u"°C"  |> u"K"  # given  
T2 = 300u"°C" |> u"K"  # given

W_out = 0.5*(P1 + P2)*m*R_CO₂*(T2/P2 - T1/P1) |> u"kJ"
@show W_out;

W_out = -250.2448267561042 kJ


----

3. Find $Q_{in}$  

  Employ the First Law, as usual:

  \begin{array}{c}
    E_{in} &-& E_{out} &=& \Delta E_{sys} \\
    Q_{in} &-& W_{out} &=& m \big( u_2 - u_1 \big) \\{}
  \end{array}

  $$ Q_{in} = W_{out} + m \big( u_2 - u_1 \big)$$

In [3]:
Q_in = W_out + m*(u2 - u1)
@show Q_in;

Q_in = -37.40774427484993 kJ


<div class="mcs-bs-callout mcs-bs-callout-success">

#### Found
  
1. $ \pu{Q_{in} = -37.408 kJ} $
1. $ \pu{W_{out} = -250.245 kJ} $

</div>

<footer class="psy"><div class="psy">ψᵟ</div></footer>